crates/common/src/compile.rs

```diff
@@ -401,7 +405,7 @@ pub fn compile_target_with_filter(
 /// Creates and compiles a project from an Etherscan source.
 pub async fn compile_from_source(
     metadata: &Metadata,
-) -> Result<(ArtifactId, ContractBytecodeSome)> {
+) -> Result<(ArtifactId, u32, ContractBytecodeSome)> {
```

Comment:
this needs docs that this returns the fileid of the solidity artifact.

Reply:
Done, thanks!

Reply:
Done thanks!